sdk/python/lib/pulumi/runtime/proto/status_pb2.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Are these expected?

Reply:
I don't know, but our build system runs protoc is inside a docker container so I don't think it's a thing from my machine specifically.

Reply:
Is there any objection to keeping these? We probably should have them anyway.